mmaction/models/backbones/cuda_shift/setup.py

```diff
@@ -1,9 +1,24 @@
 import os
+import subprocess
 from setuptools import find_packages, setup
 
 import torch
 from torch.utils.cpp_extension import CUDAExtension
 
+
+def _find_cuda_home():
+    # guess rule 3 of torch.utils.cpp_extension
+    nvcc = subprocess.check_output(['which', 'nvcc']).decode().rstrip('\r\n')
+    cuda_home = os.path.dirname(os.path.dirname(nvcc))
+    print(f'find cuda home:{cuda_home}')
+    return cuda_home
+
+
+# remember to overwrite PyTorch auto-detected cuda home which
```

Comment:
what's the reason?

Reply:
Because PyTorch determines the `CUDA_HOME` using this priority: `os.environ['CUDA_HOME']` > path to `CUDA` supporting `nvcc` > `'/usr/local/cuda'`, although the first two guess should be the same in most cases. Here, it prefers using the second guess.

Reply:
okay, not sure why this matters